definitions/procedures/backup/offline/foreman_db.rb

```diff
@@ -31,21 +31,42 @@ def run
       private
 
       def local_backup
-        with_spinner("Collecting data from #{pg_data_dir}") do
-          feature(:foreman_database).backup_local(
-            pg_backup_file,
-            :listed_incremental => File.join(@backup_dir, '.postgres.snar'),
-            :volume_size => @tar_volume_size,
-            :data_dir => pg_data_dir
-          )
+        with_spinner("Collecting data from #{pg_data_dir.join(',')}") do
+          pg_data_dirs.each_with_index do |pg_dir, index|
+            do_backup(pg_dir, 'create') if index == 0
+            do_backup(pg_dir, 'append') if index != 0
```

Comment:
Personally I'm not a big fan of using guards this way and would expand it. My main reason is that it's much easier (at least to me) to follow the code flow when reading it.
```suggestion
            if index == 0
              do_backup(pg_dir, 'create') 
            else
              do_backup(pg_dir, 'append')
            end
```